packages/components/src/composer/composer.tsx

```diff
@@ -50,7 +50,7 @@ const Composer = (props: Props) => {
     <BlurView
       intensity={40}
       style={{
-        borderRadius: 20,
+        borderRadius: '$20',
```

Comment:
Typescript says: Types of property 'borderRadius' are incompatible. (string vs. number) How do we want to bypass it?

Reply:
BlurView is (obviously) not hooked up to Tamagui theme. The answer here is probably to extract expo-blur to our separate component, which is hooked up to theme using `styled`. But I don't mind keeping it hard-coded here, for the sake of simplicity, for now.

Reply:
Similarly what is done to Image.